docs/source/hsm.md

```diff
@@ -53,11 +53,12 @@ bccsp:
     Label: fabric
     hash: SHA2
     security: 256
+    Immutable: false
 ```
 
-You can also use environment variables to override the relevant fields of the
-configuration file. If you are connecting to an HSM using the Fabric CA server,
-you need to set the following environment variables:
+By default, when private keys are generated using the HSM, the private key is mutable, meaning PKCS11 private key  attributes can be changed after the key is generated. Setting `Immutable` to `true` means that the private key attributes cannot be altered after key generation. Before you configure immutability by setting `Immutable: true`, ensure that PKCS11 object cloning is supported by the HSM.
+
```

Comment:
it may be worth mentioning what `PKCS11 object cloning` means, since I could not find any detail with a quick search.

Reply:
Cloning is same copy object as C_CopyObject Covered under Object Management Section 5.7.2 of pkcs11 interface document https://docs.oasis-open.org/pkcs11/pkcs11-base/v3.0/csprd01/pkcs11-base-v3.0-csprd01.pdf. Do you think `clone` should be replaced by` copy` @nikhil550 ?

Reply:
that is definitely a lot easier to find.

Reply:
Thanks @ashku I changed `cloning` to `copy`